pkg/oomstore/types/config.go

```diff
@@ -26,6 +26,15 @@ type OomStoreConfig struct {
 	MetadataStore MetadataStoreConfig `yaml:"metadata-store"`
 	OfflineStore  OfflineStoreConfig  `yaml:"offline-store"`
 	OnlineStore   OnlineStoreConfig   `yaml:"online-store"`
+
+	StreamPushProcessor *StreamPushProcessorConfig `yaml:"stream-push-processor"`
+}
+
+type StreamPushProcessorConfig struct {
+	RevisionInterval time.Duration `yaml:"revision-interval"`
```

Comment:
`RevisionInterval` -> `SnapshotInterval` ?

Reply:
The `pushStreamProcessor` only controls the revision, the `oomcli snapshot` checks the revision to decide whether to create a new snapshot, so it seems more appropriate to call it revision here 🤔

Reply:
`RevisionInterval` looks like it applies to batch features as well 🤣

Reply:
`StreamRevisionInterval`?

Reply:
@lianxmfor This interval is actually for the snapshot service, IMO `SnapshotInterval` is a appropiate name.

Reply:
@wfxr make sense

Reply:
you can understand it this way: We use `SnapshotInterval` to determin when to create a new revision.